tests/neg/overrides.scala

```diff
@@ -34,11 +34,6 @@ package p2 { // all being in the same package compiles fine
     }
   }
 
-  abstract class T3 extends T2 {
-    class A {   // error: classes cannot be overridden
-      bug()
-    }
-  }
 }
```

Comment:
It now masks the other errors because the error is raised now in typer, whereas the others would be raised in RefChecks.